lib/bootparams.js

```diff
@@ -430,6 +434,20 @@ function getBootParams(opts, callback) {
                     fArg.platforms = platforms;
                     cb();
                 });
+            },
+
+            function _getNodeConfig(fArg, cb) {
+                mod_node_config.getNodeConfig(opts, function getCfgCb(getErr) {
+                    if (getErr) {
+                        log.error(getErr, util.format(
+                            'Error getting %s file', mod_node_config.fileName));
+                        cb(getErr);
+                        return;
+                    }
+                    fArg.nodeconfig = util.format('%s/%s/%s', opts.tftpRoot,
```

Comment:
I not sure who uses this `fArg.nodeconfig` variable? It's returned in the getBootParams() callback, but I still don't see it referenced anywhere after that?

Reply:
> I think this will work, but it would be much simpler if we just `lofs` mount the node.config into the booter zone - like we already do with the os, i.e.
> https://github.com/joyent/sdc-headnode/blob/master/config/sapi/services/dhcpd/service.json#L10
> 
> Then the booter zone just needs to add the url path to node.config ?

Well, the idea was to avoid adding another level of coupling (assuming we may work on HA at some point) but given how easy it would be to workaround this in the future, I can give that a try, for sure).

Reply:
> I not sure who uses this `fArg.nodeconfig` variable? It's returned in the getBootParams() callback, but I still don't see it referenced anywhere after that?

If we take the path through lofs mount, all this file will be removed so ... ignoring for now :-)